index.html

```diff
@@ -40,31 +51,222 @@ <h1 class="heading_second">
+          <ul class="tech_list">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="btn1" type="button">See Project</button>
+        </div>
+      </div>
+    
+    </div>
+ 
+
+
+    <section id="secondlast_section">
+      <div class="section-last-conatiner">
+        <div class="home_containe">
+          <div>
+            <h1 class="aboutdiv">About<br>Myself</h1>
+          </div>
+          <div class="first-para">
+          <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my
+            work and experience! If you like what you see and have a project you need coded, don't hestiate to contact
+            me.
+          </p>
+        </div>
+          <span>LET'S CONNECT</span>
+            <ul class="social_links">
+            <li>
+              <a href="#">
+                <img src="./images/Vector-2.jpg" class=logo-icons alt=union>
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/Linkedin-icon.jpg" class=logo-icons alt=union>
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/Vector-3.jpg" class=logo-icons alt=union>
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/Vector-4.jpg" class=logo-icons alt=union>
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/Vector.jpg" class=logo-icons alt=union>
+              </a>
+            </li>
+          </ul>
+          <button class="btn1" type="button">Get My Resume</button>
+        </div>
+        <div>
+          <div>
+            <div class="languages_logo">
+              <h4 class="languages_heading">Languages</h4>
+              <div class="image_float">
+                <img src="./images/IconChevronDownLarger.png" alt="icon">
+              </div>
+            </div>
+            <ul class="languages">
+              <li class="ellipse1">
+                <img src="./images/Ellipse_j.png" alt="icon">
+                <span>JavaScript</span>
+              </li>
+              <li class="ellipse1">
+                <img src="/images/Ellipse_h.png" alt="icon">
+                <span>CSS</span>
+              </li>
+              <li class="ellipse1">
+                <img src="./images/Ellipse_.png" alt="con">
+                <span>HTML</span>
+              </li>
+            </ul>
+          </div>
+          <div class="languages_frameworks">
+            <h4>Frameworks</h4>
+            <span>&#62;</span>
+          </div>
+          <hr>
+          <div class="languages_frameworks">
+            <h4>Skills</h4>
+            <span>&#62;</span>
+            </div>
+          <hr>
```

Comment:
- [ ] You are yet to implement the chevron design as requested earlier. Kindly do that.

  ![chev-r](https://user-images.githubusercontent.com/36542986/164947975-79f997d6-3d71-45e6-ad03-1103005e23bf.jpg)

Reply:
thanks, please check